plugins/catalog-backend-module-incremental-ingestion/README.md

```diff
@@ -175,17 +175,17 @@ import { IncrementalEntityProvider } from '@backstage/plugin-catalog-backend-mod
 
 // This will include your pagination information, let's say our API accepts a `page` parameter.
 // In this case, the cursor will include `page`
-interface MyApiCursor {
+interface Cursor {
```

Comment:
This change is not necessary:
```suggestion
interface MyApiCursor {
```
What's originally inconsistent in the docs is the one on line 200.

Reply:
Sure. All I was saying is that the only inconsistent mention is on line 200, where it says `Cursor` and `Context`. Every other place is using `MyApiCursor` and `MyContext`. So, instead of making all those changes, we could just change line 200.